core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -248,9 +248,12 @@ private[spark] class SecurityManager(sparkConf: SparkConf)
   // configuration at a specified namespace. The namespace *must* start with spark.ssl.
   val fileServerSSLOptions = SSLOptions.parse(sparkConf, "spark.ssl.fs", Some(defaultSSLOptions))
   val akkaSSLOptions = SSLOptions.parse(sparkConf, "spark.ssl.akka", Some(defaultSSLOptions))
+  val dispatcherSSLOptions =
+    SSLOptions.parse(sparkConf, "spark.ssl.dispatcher", Some(defaultSSLOptions))
```

Comment:
It's used here: https://github.com/mesosphere/spark/pull/5/files#diff-0e5e426025e1e8a0d1dd706bca3c62a7R83

We can ret rid of the dispatcher term. Maybe, `submissionServer` instead?